### ISSUE TITLE
Do you want the packages

### ISSUE THREAD

**Issue by AlexMeah** [HFE/boilerplate#24]
@mikefosbrook ?

**Comment by marvin-speakman** [comment]
Is this a hostage situation ...

**Comment by AlexMeah** [comment]
![image](https://cloud.githubusercontent.com/assets/3817298/7495673/4a71fdb4-f407-11e4-864b-72680ddaaa65.png)

**Comment by AlexMeah** [comment]
No I vaguely remember a drunken chat with @mikefosbrook, doesn't make sense for them to be my name.

**Comment by marvin-speakman** [comment]
LOLZ, that @mikefosbrook is a bad influence and always drunk!

**Comment by mikefosbrook** [comment]
Hi guys. I thought this was an old post for a minute. It's been a bit tricky trying to keep on top of all the boilerplate stuff since we all saw each other last in El Capo. 

While you're on can one of you make me and Ruth owners on this account. Also, no offence but we'll need to take you off the account eventually :s

Regarding the Node Modules/Packages that's a funny one - I managed to put them on our Stash account and they work but if you're offering it might be easiest to just hand over the packages to me so we can work with them as usual with the added bonus of being able to make modifications?

The ones we would want to hide from the world are:
https://www.npmjs.com/package/grunt-walrus
https://www.npmjs.com/package/grunt-irep-assets
https://www.npmjs.com/package/grunt-zip-directories
https://www.npmjs.com/package/grunt-capturelocal
https://www.npmjs.com/package/grunt-lux

**Comment by AlexMeah** [comment]
Yeah can give you them all what, is your username on npm? Also what is Ruth's github username?

**Comment by mikefosbrook** [comment]
Username on NPM is called hlfrontend
Ruth's Github account is ruthh

Cheers

**Comment by mikefosbrook** [comment]
@AlexMeah I tried getting in touch with you via contact form on your website. How do I update the docs on http://hfe.github.io/boilerplate/ ? I've followed the instructions on the github website

I run this command in the terminal:

git clone https://github.com/HFE/hfe.github.io/

It then asks for username and password but it doesn't accept my username and password. What username and password should I be using?

Also, although you've given me access

**Comment by mikefosbrook** [comment]
I worked it out - github pages exist as a branch of the boilerplate repo called gh-pages.

**Comment by AlexMeah** [comment]
@mikefosbrook Hi mike, forgot to mention you should have all the packages now check your npm :)

**Comment by AlexMeah** [comment]
You can get me on alexmeah at gmail.com
